.coderabbit.yaml

```
@@ -1,18 +1,26 @@
   review_status: true
   collapse_walkthrough: false
   auto_review:
     enabled: true
     drafts: false
     base_branches:
+      - develop
       - develop
       - main
-chat:
-  auto_reply: true
+  path_filters: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)